### PR TITLE
Fix bagel for Windows users

### PR DESCRIPTION
**Pull request by nopcoder**
fix docker build to include entrypoint.sh as unix line format

### CODE REVIEW

**Review by itaiad200 — COMMENTED** [review]
Checking this and approving if it works :)

File: deployments/compose/docker-compose.yml [hunk 1]
```diff
@@ -182,7 +182,12 @@ services:
 
   spark-thrift:
     image: docker.io/bitnami/spark:3
-    command: ["bash","-c", "/opt/bitnami/entrypoint.sh"]
+    # Windows bug (see https://github.com/treeverse/lakeFS/pull/2871)
+    # Need to copy the mounted file and replace the line endings for Windows hosts.blkio_config:
+    # For linux machines, this is a no-op
+    command: ["bash","-c", "cp /opt/bitnami/entrypoint.sh /opt/bitnami/entrypoint-copy.sh &&
```
Comment (nopcoder) [inline deployments/compose/docker-compose.yml:188]: prefer not to enable this hack - I assume that volume mount should work if the file format is kept

Comment (arielshaqed) [reply deployments/compose/docker-compose.yml:188]: Would it work if we created a new Dockerfile that started with docker.io/bitnami/spark:3 and edited that file, creating a new good Docker image?

**Review by arielshaqed — COMMENTED** [review]
*WARNING*.  :warning:  Potentially stupid suggestion.

**Review by itaiad200 — APPROVED** [review]
It works! amazing work!

**Pull request merged**